app/controllers/response_sets_controller.rb

```diff
@@ -33,6 +33,30 @@ def create
   def publish
     if @response_set.status == 'draft'
       @response_set.publish
+      respond_to do |format|
+        format.html { redirect_to @response_set, notice: 'Draft question published' }
```

Comment:
We should eliminate all html responses. My guess is that this will just error as there shouldn't be an erb template for it to render.